nova/compute/manager.py

```diff
@@ -10680,6 +10681,35 @@ def _reclaim_queued_deletes(self, context):
                                 "instance: %s",
                                 e, instance=instance)
 
+    @periodic_task.periodic_task(
+            spacing=CONF.instances_stuck_in_deleting_cleanup_interval)
+    def _cleanup_instances_stuck_in_deleting(self, context):
+        """Reclaim instances stuck in DELETING state, but not really destroyed,
```

Comment:
What does "Reclaim" mean here? Maybe better "Retry terminate_instances for instances stuck in DELETING state"

Reply:
fixed